packages/containers/src/ActionBar/ActionBar.connect.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+
+import Container from './ActionBar.container';
+
+export function mapDispatchToProps(dispatch) {
+	return {
+		dispatch,
```

Comment:
The dispatch props is not used in container.
So if we remove that, we can remove the connector